data-reporting-service/investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java

```diff
@@ -44,10 +51,39 @@ public InvestigationTransformed transformInvestigationData(Investigation investi
         transformNotificationIds(investigation.getObservationNotificationIds(), objectMapper);
```

Comment:
Do we still need processing notification ids and `nrt_investigation_notification`?

Reply:
for now, yes - https://cdc-nbs.atlassian.net/wiki/spaces/NM/pages/533790760/005+investigation#Table---investigation_notification-pHASE-OUT but we need to remove it once a decision is made